spring-integration-core/src/main/java/org/springframework/integration/expression/ExpressionUtils.java

```diff
@@ -168,29 +172,49 @@ private static EvaluationContext createEvaluationContext(@Nullable ConversionSer
 	public static File expressionToFile(Expression expression, EvaluationContext evaluationContext,
 			@Nullable Message<?> message, String name) {
```

Comment:
We should take this opportunity to improve the javadocs for the `name` parameter.

`@param name the name of the result of the evaluation.`

`@param name the name of the property we are evaluating an expression for` ?